src/java/us/kbase/templates/module_test_java_client.vm.properties

```diff
@@ -189,6 +192,10 @@ public class ${java_module_name}ServerTest {
         // Check returned data with
         // Assert.assertEquals(..., ret.getSomeProperty());
         // ... or other JUnit methods.
+        Map<String, UObject> params = new HashMap<>();
+        params.put("workspace_name", new UObject(getWsName()));
+        params.put("parameter_1", new UObject("Hello World"));
+        final ReportResults ret = impl.runJavafoo(params, token, getContext());
```

Comment:
I think this should be something like `impl.runTestJavaHelloWorld(...`

The java tests don't pass for me as it is now.

Reply:
Changing this to `impl.runTestJavaHelloWorld` seems to fix tests